analyser/main.py

```
@@ -35,40 +32,209 @@
+        raise Exception(
+            f"{action}: backend returned status {res.status_code} with "
+            f"non-JSON body (length {len(res.text or '')}): {body_excerpt!r}"
+        )
+
+
+def get_next_analysis_tracks(model='discogs_multi_embeddings-effnet-bs64-1', purchased=True, batch_size=10):
     print("Getting next tracks to analyse")
     print(purchased)
-    print(f"{os.getenv('API_URL')}/admin/analyse?model={model}&batch_size={batch_size}&purchased={'true' if purchased else ''}")
+    print(f"{get_api_url()}/admin/analyse?model={model}&batch_size={batch_size}&purchased={'true' if purchased else ''}")
     res = requests.get(
```

Ruff (0.15.14)

[error] Probable use of `requests` call without timeout

(S113)